x-pack/plugins/lens/public/visualizations/xy/visualization.tsx

```diff
@@ -618,6 +618,7 @@ export const getXyVisualization = ({
               appName: 'lens',
               storage,
               uiSettings: core.uiSettings,
+              settings: core.settings,
```

Comment:
I think that the changes on the xy visualizations file are not needed. Do you have failures without adding this? 🤔